scripts/misc/grant_access_to_collab.sh

```diff
@@ -129,10 +134,11 @@ prefix="${prefix%/}"
 ################################################################################
 for t in "${file_types[@]}";
 do
-    echo -e "\nGranting user ${user_email} with ${access_level}-level access to file type ${t} under gs://${gcs_bucket}/${prefix}/"
-    gsutil -mq acl \
-        ch -u "${user_email}":"${access_level}" \
-        -R "gs://${gcs_bucket}/${prefix}/**${t}"
+    echo -e "\nGranting ${account_email} with ${access_level}-level access to file type ${t} under gs://${gcs_bucket}/${prefix}/"
+    gsutil -mq acl ch \
+        "${account_type}" \
+        "${account_email}":"${access_level}" \
+        -R "gs://${gcs_bucket}/${prefix}/**/*${t}"
```

Comment:
why the extra `/*` ?

Reply:
That double star is for any "sub-directories".
That single star is for globing the file with the right extension.

I've tested this by checking for BAMs that are two levels below `prefix`, and immediately under `prefix`.